src/components/content-tab-import-export.tsx

```diff
@@ -21,15 +22,34 @@ interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
 
-export const ExportSite = ( { selectedSite }: { selectedSite: SiteDetails } ) => {
+export const ExportSite = ( {
+	selectedSite,
+	isAnySiteSyncing,
+	isThisSiteSyncing,
+}: {
+	selectedSite: SiteDetails;
+	isAnySiteSyncing: boolean;
+	isThisSiteSyncing: boolean;
+} ) => {
 	const { exportState, exportFullSite, exportDatabase, importState } = useImportExport();
 	const { [ selectedSite.id ]: currentProgress } = exportState;
-	const isSiteImporting = importState[ selectedSite.id ]?.progress < 100;
-	const { isAnySitePulling, isAnySitePushing } = useSyncSites();
-	const isExportDisabled = isSiteImporting || isAnySitePulling || isAnySitePushing;
-	const siteNotReadyForExportMessage = __(
-		'This site is being imported. Please wait for the import to finish before you export it.'
-	);
+	const isImporting = importState[ selectedSite.id ]?.progress < 100;
+	const isExportDisabled = isImporting || isAnySiteSyncing;
+
+	let tooltipText;
```

Comment:
If the tooltip text isn't provided, the Tooltip component will make sure the tooltip gets hidden: https://github.com/Automattic/studio/pull/722 (PR in progress).